src/amp-shadow.js

```diff
@@ -60,9 +60,10 @@ adoptShadowMode(self);
 // tag to give some information that can be used in error reports.
 // (At least by sophisticated users).
 if (self.console) {
+  const versionString = humanRtv(getMode(self).rtvVersion);
   (console.info || console.log).call(
     console,
-    `Powered by AMP ⚡ HTML shadows – Version ${internalRuntimeVersion()}`
```

Comment:
Why was "shadows" removed? Copy-pasta? I think it should be left as is, in case someone relies on that string while processing logs from `amp-shadow.js`.

Reply:
That is correct; fixed.